test/integration/api.bats

```diff
@@ -508,12 +569,39 @@ function teardown() {
 	skip
 }
 
-# FIXME
 @test "docker version" {
-	skip
+	start_docker 3
+	swarm_manage
+
+	# version
+	run docker_swarm version
+	[ "$status" -eq 0 ]
+	[ "${#lines[@]}" -ge 8 ]
+
+	# verify
+	client_reg='^Client version: [0-9]+\.[0-9]+\.[0-9]+$'
+	server_reg='^Server version: swarm/[0-9]+\.[0-9]+\.[0-9]+$'
+	[[ "${lines[0]}" =~ $client_reg ]]
+	[[ "${lines[5]}" =~ $server_reg ]]
 }
 
-# FIXME
 @test "docker wait" {
-	skip
+	start_docker 3
+	swarm_manage
+	
+	# run after 10 seconds, test_container will exit
+	run docker_swarm run -d --name test_container busybox sleep 10
```

Comment:
10 is way too long (we don't want to slow down the tests).

3 seconds should be plenty